src/main/java/org/broadinstitute/hellbender/utils/SimpleInterval.java

```diff
@@ -9,7 +9,7 @@
  *
  *@warning 0 length intervals are NOT currently allowed, but support may be added in the future
  */
-public class SimpleInterval implements Locatable {
+public final class SimpleInterval implements Locatable {
```

Comment:
This class is actually intended to serve as a base class for interval classes -- should not be final.